Sources/Lifecycle/Lifecycle.swift

```diff
@@ -18,9 +18,9 @@ import Darwin
 import Glibc
 #endif
 import Backtrace
+import CoreMetrics
```

Comment:
is it intentional? AFAIR libraries shouldn't normally depend on CoreMetrics unless implementing a MetricFactory.

Reply:
CoreMetrics is useful when you do not want to pul in Foundation